chain-tx-enclave-next/enclave-ra/ra-enclave/src/context.rs

```diff
@@ -40,7 +42,10 @@ impl EnclaveRaContext {
     }
 
     /// Returns current certificate. If current certificate is no longer valid, then it creates a new one
-    pub fn get_certificate(&self) -> Result<Certificate, EnclaveRaContextError> {
+    pub fn get_certificate(
+        &self,
+        expiration_time: Option<Duration>,
```

Comment:
Can you add `certificate_expiration_time` in `EnclaveRaConfig` and set it in a struct member instead of taking this as function argument (similar to `validity_duration`)?